app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1143,19 +1160,21 @@ class PipelineSampleReport extends React.Component {
                           </span>
                         Taxonomy
                         </th>
-                        {this.render_column_header('', 'Score', 'NT_aggregatescore', 'Aggregate score: ( |genus.NT.Z| * species.NT.Z * species.NT.rPM ) + ( |genus.NR.Z| * species.NR.Z * species.NR.rPM )') }
-                        {this.render_column_header('NT', 'Z', 'NT_zscore', 'Z-score relative to background model for alignments to NCBI NT') }
-                        {this.render_column_header('NT', 'rPM', 'NT_rpm', 'Number of reads aligning to the taxon in the NCBI NT database per million total input reads')}
-                        {this.render_column_header('NT', 'r', 'NT_r', 'Number of reads aligning to the taxon in the NCBI NT database')}
-                        {this.render_column_header('NT', '%id', 'NT_percentidentity', 'Average percent-identity of alignments to NCBI NT')}
-                        {this.render_column_header('NT', 'log(1/E)', 'NT_neglogevalue', 'Average log-10-transformed expect value for alignments to NCBI NT')}
-                        {this.render_column_header('NT', '%conc', 'NT_percentconcordant', 'Percentage of aligned reads belonging to a concordantly mappped pair (NCBI NT)')}
-                        {this.render_column_header('NR', 'Z', 'NR_zscore', 'Z-score relative to background model for alignments to NCBI NR') }
-                        {this.render_column_header('NR', 'rPM', 'NR_rpm', 'Number of reads aligning to the taxon in the NCBI NR database per million total input reads')}
-                        {this.render_column_header('NR', 'r', 'NR_r', 'Number of reads aligning to the taxon in the NCBI NR database')}
-                        {this.render_column_header('NR', '%id', 'NR_percentidentity', 'Average percent-identity of alignments to NCBI NR')}
-                        {this.render_column_header('NR', 'log(1/E)', 'NR_neglogevalue', 'Average log-10-transformed expect value for alignments to NCBI NR')}
-                        {this.render_column_header('NR', '%conc', 'NR_percentconcordant', 'Percentage of aligned reads belonging to a concordantly mappped pair (NCBI NR)')}
+                        {this.render_column_header('Score', `NT_aggregatescore`, 'Aggregate score: ( |genus.NT.Z| * species.NT.Z * species.NT.rPM ) + ( |genus.NR.Z| * species.NR.Z * species.NR.rPM )') }
+                        {this.render_column_header('Z', `${this.state.countType}_zscore`, `Z-score relative to background model for alignments to NCBI ${this.state.countType}`) }
+                        {this.render_column_header('rPM', `${this.state.countType}_rpm`, `Number of reads aligning to the taxon in the NCBI ${this.state.countType} database per million total input reads`)} 
```

Comment:
Since both values are shown (albeit with different emphasis), I think you can remove the reference to `his.state.countType` from the tooltip

Reply:
Okay, do you suggest we make the tooltip messages static then? so instead we 
say `Number of reads aligning to the taxon in the NCBI NT/NR database per million total input reads

Reply:
yes!